package.json

```diff
@@ -2,14 +2,13 @@
   "private": true,
   "scripts": {
     "clean": "rimraf output && rimraf .pulp-cache",
-    "build": "jshint src && jscs src && pulp build -- --censor-lib --strict",
+    "build": "eslint src && pulp build -- --censor-lib --strict",
     "test": "pulp test"
```

Comment:
There isn't a test for this repo. Should we add a placeholder one or remove this line?